code/game/machinery/_machinery.dm

```diff
@@ -336,44 +336,52 @@
 	if((machine_stat & (NOPOWER|BROKEN)) && !(interaction_flags_machine & INTERACT_MACHINE_OFFLINE)) // Check if the machine is broken, and if we can still interact with it if so
 		return FALSE
 
-	var/silicon = issilicon(user)
-	if(panel_open && !(interaction_flags_machine & INTERACT_MACHINE_OPEN)) // Check if we can interact with an open panel machine, if the panel is open
-		if(!silicon || !(interaction_flags_machine & INTERACT_MACHINE_OPEN_SILICON))
-			return FALSE
+	if(isAdminGhostAI(user))
+		return TRUE //if you're an admin, you probably know what you're doing (or at least have permission to do what you're doing)
```

Comment:
![aoKAQlDW_700wa_0](https://user-images.githubusercontent.com/42839747/111229163-baa03a80-85e5-11eb-91f6-63cd446f1415.gif)
admins know what they doing

at least LOG it so that if an admin's feeling funny we can know about this instead of blaming not working code

Reply:
you want me to send a logging message any time an admin does anything as an admin ghost

Reply:
wouldn't the admins report that they were in admin ghost AI mode, instead of that they were a normal AI

Reply:
anything on a machine since they can interact with it and change the machine UI settings, so yes